### ISSUE TITLE
The slidetoact appears on top of other views which are higher in the hierarchy

### ISSUE THREAD

**Issue by yoninezer** [cortinico/slidetoact#17]
If Im adding a match parent width & height container to to bottom of the activity - the slidetoact view still appears, even though it is lower in the hierarchy. any thoughts?

**Comment by cortinico** [comment]
Hey @yoninezer 
Can you be more specific? Also code/screenshots would be better to understand.

**Comment by yoninezer** [comment]
Hey @cortinico.  Sorry for not being clear enough. I'll try again :)

Let's say I have an activity layout which are the following:
`<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="match_parent">
    
    <com.ncorti.slidetoact.SlideToActView
        android:id="@+id/event_slide"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:elevation="6dp"
        app:area_margin="10dp"
        app:inner_color="#F5A623"
        app:outer_color="@android:color/black"
        app:slider_height="104dp"
        app:slider_locked="false"
        app:text="Flare"
        app:text_size="30sp" />

    <TextView
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_centerInParent="true"
        android:text="Test"
        android:textColor="#ffffff" />

    <LinearLayout
        android:id="@+id/shader"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:background="@android:color/black"
        android:orientation="horizontal" />
</RelativeLayout>`

So basically, the SlideToActView is still visible, even though I have a huge LinearLayout over it (id=shader).

And as you can see, as expected, the TextView I put is not shown, I would expect for that same behavior from the SlideToActView (I guess).
Is there any way I can change the "on-top" priority?

<a href="http://tinypic.com?ref=28im5cj" target="_blank"><img src="http://i63.tinypic.com/28im5cj.png" border="0" alt="Image and video hosting by TinyPic"></a>


Thanks in advanced

**Comment by cortinico** [comment]
Hey @yoninezer 
Thanks for the code. So basically this is caused by the `6dp` elevation that you added to the `SlideToActView`. This is causing the z-ordering of your child views to be different than the order of your views in the layout file.

That's the intended behavior so I'll close here since is not a bug.
To fix your problem, either remove the elevation from the `SlideToActView` or add `6dp` elevation or more to your `LinearLayout`.

Hope I solved your problems :) and thanks again for using the library.

**Comment by yoninezer** [comment]
Worked like a charm 👍  
Much thanks !
